code/modules/jobs/job_types/roguetown/nobility/lord.dm

```diff
@@ -341,7 +341,7 @@ GLOBAL_LIST_EMPTY(lord_titles)
 	if(QDELETED(recruit) || QDELETED(recruiter))
 		return FALSE
 	if(HAS_TRAIT(recruit, TRAIT_NOBLE))
-		if(!HAS_TRAIT(recruit,TRAIT_OUTLANDER))
+		if(!(recruit.job in list("Adventurer", "Mercenary", "Bandit", "Wretch","Inquisitor","Suitor","Orthodoxist","Migrant")))
```

Comment:
you probably want to define this as a list elsewhere so it could be referenced more consistently / easier